templateflow/templateflow_get.py

```diff
@@ -0,0 +1,197 @@
+    parser = argparse.ArgumentParser(description='TemplateFlow templates retriever.')
+
+    r_group = parser.add_argument_group('retrieve options')
+    r_group.add_argument('-r', '--resolution', 
+                        action='store',
+                        default='unset',
+                        help='Index to an specific spatial resolution of the template, multiple values coma separated may be given')
+
+    r_group.add_argument('-s', '--suffix', 
+                        action='store',
+                        default='unset',
+                        help='BIDS suffix')
+
+    r_group.add_argument('-a', '--atlas', 
+                        action='store',
+                        default='unset',
+                        help='Name of a particular atlas')
+
+    r_group.add_argument('-e', '--extension', 
+                        action='store',
+                        default='unset',
+                        help='file extension')
+
+    r_group.add_argument('-H', '--hemi', 
+                        action='store',
+                        choices=['R', 'L'],
+                        default='unset',
+                        help='select Hemisphere')
+
+    r_group.add_argument('-S', '--space', 
+                        action='store',
+                        default='unset',
+                        help='Space template is mapped to')
+
+    r_group.add_argument('-d', '--density', 
+                        action='store',
+                        default='unset',
+                        help='Surface density')
+    
+    r_group.add_argument('-D', '--desc', 
+                        action='store',
+                        default='unset',
+                        help='Description field')
+
+    r_group.add_argument('-c', '--cohort', 
+                        action='store',
+                        default='unset',
+                        help='Template cohort')
+
+    r_group.add_argument('-L', '--label', 
+                        action='store',
+                        default='unset',
+                        help='Template label')
+
+    i_group = parser.add_argument_group('info environement')
+    i_group.add_argument('-l', '--list', 
+		    action='store_true',
+		    default=False,
+		    help='list available local templates')
+
+    i_group.add_argument('-m', '--metadata', 
+		    action='store_true',
+		    default=False,
+		    help='Get full metadata for given template')
+
+    i_group.add_argument('-C', '--citations', 
+		    action='store_true',
+		    default=False,
+		    help='Get template citations')
+
+    i_group.add_argument('-B', '--bibtex-citations', 
+		    action='store_true',
+		    default=False,
+		    help='Get template citation in bibtex-citations')
+
+    m_group = parser.add_argument_group('misc options')
+    m_group.add_argument('-o', '--tf-cache', 
+		    action='store',
+		    default='unset',
+		    help='Directory to save templates in. default `${HOME}/.cache/templateflow`')
```

Comment:
this works around the `TEMPLATEFLOW_HOME` environment variable definition. please remove the option to do that.

Reply:
not sure to understand 
this will allow users to grab the templates where they want.
maybee I did not cactch the templateflow philosophy

Reply:
OK got the point. I will adapt regarding this point. 
thanks for the feedback

Eric